README.md

```diff
@@ -20,15 +20,16 @@ This Minetest mod offers changeable player skins with a graphical interface for
 
 ## Installing skins
 
-### Download from the [database](http://minetest.fensta.bplaced.net/)
+### Download from the [database](https://skinsdb.terraqueststudios.net/)
 
 #### Ingame Downloader
 
 1) Get Minetest 5.1.0-dev-cb00632 or newer
-2) Start your world
-3) Run `/skinsdb_download_skins <skindb start page> <amount of pages>`
-4) Wait for the Minetest server to shut down
-5) Start the server again
+2) In the settings menu show advanced options, find the "Developer Options" tab and add "skinsdb" to "Trusted mods"
```

Comment:
What about server builds? This instruction only covers installations that have a GUI.

Reply:
Thanks. Added